contracts/BodhiToken.sol

```diff
@@ -81,6 +81,8 @@ contract BodhiToken is StandardToken, Ownable {
   }
 
   function mintReservedTokens(uint256 _amount) onlyOwner {
+    require(totalSupply + _amount > totalSupply);
```

Comment:
what does this mean? _amount > 0?
If you mean to check totalTotalSupply, I think
line 85's `require(checkedSupply <= tokenTotalSupply);` already achieve what you are trying to do

Reply:
This is to check for overflow: https://github.com/ConsenSys/smart-contract-best-practices#integer-overflow-and-underflow

Even though unlikely in this case, its important to have the check. any uint can reset back to 0 if it hits the maximum value.